examples/sile-and-markdown.md

```diff
@@ -234,8 +234,17 @@ image syntax ---Note that any unit system supported by SILE is accepted.
 An image with nonempty caption (i.e. "alternate" text), occurring alone by itself in a paragraph,
 will be rendered as a figure with a caption. If your class or previously loaded packages
 provide a `captioned-figure` environment, it will be wrapped around the image (and it is then assumed to
-take care of a `\caption` content, i.e. to extract and display it appropriately).  Otherwise,
-the converter uses its own fallback method.
+take care of a `\caption` content, i.e. to extract and display it appropriately).
+ Otherwise, the converter uses its own fallback method.
+
+### Maths
+
+TeX-like math between `$` ("inline mode") or `$$` ("display mode") decently works.
+Note that $20,000 and $30,000 don't parse as math, while $e^{i\pi}=-1$ does.
+There is an important constraint, though: you have to restrict yourself to the syntax subset supported
+by SILE. This being said, some nice fomulas may be achieved:
+
+$$\pi=\sum_{k=0}^\infty\frac{1}{16^k}(\frac{4}{8k+1} \minus \frac{2}{8k+4} \minus \frac{1}{8k+5} \minus \frac{1}{8k+6})$$
```

Comment:
(Aside note) Check the example... GitHub doesn't complain but `\minus` doesn't seem legit:
$$\pi=\sum_{k=0}^\infty\frac{1}{16^k}(\frac{4}{8k+1} \minus \frac{2}{8k+4} \minus \frac{1}{8k+5} \minus \frac{1}{8k+6})$$

I'm bad at TeX math :p Pandoc to HTML complains ("Could not convert TeX math"), even with `-` instead of that `\minus` (which SILE accepts, it but might be by mere chance lol)

Reply:
N.B. Also relates to https://github.com/sile-typesetter/sile/issues/1642 on the SILE side...

Reply:
GitHub accepts the minus sign (−):
$$\pi=\sum_{k=0}^\infty\frac{1}{16^k}(\frac{4}{8k+1} − \frac{2}{8k+4} − \frac{1}{8k+5} − \frac{1}{8k+6})$$

Due to the above-mentioned SILE behavior with unary/binary operators, I'll stick to it for now (I don't think it will be fixed in a decent time-frame)